src/shared/geometry.ts

```diff
@@ -438,6 +448,21 @@ export function grabPose2d(log: Log, key: string, timestamp: number, uuid?: stri
   ];
 }
 
+export function grabSample(log: Log, key: string, timestamp: number, uuid?: string): AnnotatedPose3d[] {
```

Comment:
For clarity, I would rename these `grabChoreoSample` and `grabChoreoSampleArray`. There are a lot of meanings of "sample" in a data visualization tool...